src/ec/packages/ec-component-social-media-share/social-media-share.story.js

```diff
@@ -9,26 +15,125 @@ import demoData from './demo/data';
 import SocialMediaShare from './ecl-social-media-share.html.twig';
 import notes from './README.md';
 
-demoData.links.forEach(link => {
-  if (link.icon) {
-    link.icon.forEach(icon => {
-      icon.path = defaultSprite; // eslint-disable-line no-param-reassign
-    });
-  }
-});
+const prepareSocialMediaShare = data => {
+  data.description = text(
+    'description',
+    data.description,
+    buttonLabels.required
+  );
+
+  data.links.forEach((link, i) => {
+    let label = buttonLabels.optional;
+    // Firt item we make it mandatory and not removable.
+    if (i === 0) {
+      label = buttonLabels.required;
+      link.label = select(
+        `links[${i}].link.label`,
+        getBrandedIconsOptions(true),
+        link.label,
+        label
+      );
+      link.path = text(`links[${i}].link.path`, link.path, label);
+      link.icon.forEach((icon, idx) => {
+        let options = getBrandedIconsOptions(false);
+        if (idx === 1) {
+          options = getBrandedIconsOptions(false, false, true);
+        }
+        icon.name = select(
+          `links[${i}].link.icon[${idx}].name`,
+          options,
+          icon.name,
+          label
+        );
+        icon.path = select(
+          `links[${i}].link.icon[${idx}].path`,
+          [defaultSprite],
+          defaultSprite,
+          label
+        );
+        icon.size = select(
+          `links[${i}].link.icon[${idx}].size`,
+          ['xl'],
+          'xl',
+          label
+        );
```

Comment:
icon_position?

Reply:
true, i wrongly assumed i could forget about the issue (coming from a recent update on the story-utils package) for now because were using the default value and instead is the opposite. So yes, i will find a way to fix the plugin ;)